### DISCUSSION TITLE
Cycle plan until 2023-10-12

### DISCUSSION THREAD

**Discussion by fredericalpers** [onOffice-Web-Org/oo-wp-plugin#639] [Announcements]
For the next cycle please work on [milestone v4.17](https://github.com/onOffice-Web-Org/oo-wp-plugin/milestone/6)

- https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/623 
- https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/558
- https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/631
- https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/633
- https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/491
- https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/634
- https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/575

The cycle period is until Tue 2023-10-12, inclusive.

For any questions, please don't hesitate to ask.

**Comment by yeneastgate** [comment]
Hi @fredericalpers, for this cycle, we still have two more issues which are not estimated yet: 634 & 575. For these two issues, we need time to try implementing first, then will estimate the time by tomorrow. 
After that, we will let you know how many small issues we can handle in this cycle. Thanks!

**Reply by yeneastgate** [reply]
@fredericalpers Because we need to handle the reviewed tasks, so you don't need to push more issues into this cycle for me. Thanks!
